cmd/skopeo/sync.go

```diff
@@ -118,6 +121,7 @@ See skopeo-sync(1) for details.
 	flags.StringVarP(&opts.destination, "dest", "d", "", "DESTINATION transport type")
 	flags.BoolVar(&opts.scoped, "scoped", false, "Images at DESTINATION are prefix using the full source image path as scope")
 	flags.StringVar(&opts.appendSuffix, "append-suffix", "", "String to append to DESTINATION tags")
+	flags.IntVar(&opts.topRefs, "top-refs", 0, "Only get amount of top largest refs after sorted by Semantic versioning")
```

Comment:
“top” is fairly vague. Maybe `--semver-latest`? (That would naturally pair with a hypothetical `--semver-since=1.7.0`)

What do others think?

---

(That would then propagate everywhere.)

Reply:
I'm struggling with top-refs too..  If we stick with it, I'd be OK with the option name of `top-refs`, but we should really spell out the other `refs` references as `references`.  FWIW, I like @mtrmac 's name suggestion.